llvm/lib/Target/AArch64/AArch64InstrInfo.td

```diff
@@ -273,6 +273,8 @@ def HasSVE2p1_or_HasSME2p1
 def HasSVE2p2orSME2p2
     : Predicate<"Subtarget->isSVEorStreamingSVEAvailable() && (Subtarget->hasSVE2p2() || Subtarget->hasSME2p2())">,
                  AssemblerPredicateWithAll<(any_of FeatureSME2p2, FeatureSVE2p2), "sme2p2 or sve2p2">;
+def NotHasSVE2p2orSME2p2
```

Comment:
It would be better for this to reflect the intent of the "feature" so perhaps something like "UseUnaryUdefPseudos" to alongside the existing UseExperimentalZeroingPseudos.